windows/test_msi.ps1

```diff
@@ -0,0 +1,56 @@
+
+param (
+    [string]$INTEGRATION="integration",
+    [string]$ARCH="amd64",
+    [string]$TAG="v0.0.0",
+    [string]$UPGRADE="false", # upgrade: upgrade msi from last released version.
+    [string]$MSI_PATH=""
+)
+
+if($UPGRADE -eq "true")
+{
+    $latest_msi_name = "nri-${INTEGRATION}-${ARCH}.msi"
+    $latest_msi_url = "https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-${INTEGRATION}/${latest_msi_name}"
+    write-host "===> Downloading latest released version of msi from ${latest_msi_url}"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    $r = Invoke-WebRequest "${latest_msi_url}" -OutFile "${latest_msi_name}"
+
+    write-host "===> Installing latest released version of msi from ${latest_msi_url}"
+    $p = Start-Process msiexec.exe -Wait -PassThru -ArgumentList "/qn /L*v msi_log /i ${latest_msi_name}"
+    if($p.ExitCode -ne 0)
+    {
+        Get-Content -Path .\msi_log
+        echo "Failed installing latest version of the msi"
+        exit 1
+    }
+}
+
+$version = $TAG.substring(1)
```

Comment:
I'd replace this by something more robust (e.g. replace "v" with ""), just so we don't run into hard to debug isses if we pass the tag without the leading `v`.

Reply:
I'm really bad at powershell but perhaps something like this would work:
```suggestion
$version = $TAG -replace "v", ""
```

Reply:
sure , was a c&p from one ps scritps in the pipeline. I will try that